sass/themes/_advice.scss

```
@@ -1,6 +1,41 @@
 @charset 'UTF-8';
 
 
+/*doc
+---
+title: "Required <select> should have first <option> with empty [value]"
+name: empty-option-on-select
+category: Advices
+---
+
+##Description
+The first child `option` element of a `select` element with a `[required]` attribute, and without a `[multiple]` attribute, and without a `[size]` attribute whose value is greater than 1, must have either an empty value attribute, or must have no text content. Consider either adding a placeholder option label, or adding a size attribute with a value equal to the number of option elements.
```

It's a bit too complex as a sentence IMHO.

```suggestion
A `<select required>` element, which isn't neither `[multiple]` nor `[size]` greater than `1`, should start with an empty `<option>`. That being said, you may use a placeholder content for this option but must ensure to use an empty `[value]` attribute; or set a `[size]` attributes to the `<select>`, which value should equal to the number of `<option>`s.
```

What's your opinion?

---

I agree, I just copied and pasted from one of the links to keep the description as close to the _official_ source, but I agree that is a bit confusing, at least for non-native English speakers like me. I struggled a bit trying to understand it at first, but your description is much better :smile: